guides/common/modules/snip_using_installer_noop.adoc

```diff
@@ -1,6 +1,6 @@
 [WARNING]
 If you customize configuration files, manually or using a tool such as Hiera, these changes are overwritten when the installation script runs during upgrading or updating.
```

Comment:
```suggestion
If you customize configuration files, manually or using a tool such as Hiera, these changes are overwritten when the maintenance script runs during upgrading or updating.
```
While we're at it.

Reply:
Agreed. I wonder why I didn't get this even if I read it. :confused: